src/zabapgit_forms.prog.abap

```diff
@@ -119,6 +119,52 @@ FORM branch_popup TABLES   tt_fields TYPE ty_sval_tt
 
 ENDFORM.                    "branch_popup
 
+FORM package_popup TABLES   tt_fields TYPE ty_sval_tt
+                   USING    pv_code TYPE clike
+                   CHANGING cs_error TYPE svale
+                            cv_show_popup TYPE c
+                   RAISING  lcx_exception ##called ##needed.
+* called dynamically from function module POPUP_GET_VALUES_USER_BUTTONS
+
+  DATA: ls_package_data TYPE scompkdtln.
+
+  FIELD-SYMBOLS: <ls_fbranch> LIKE LINE OF tt_fields.
+
+  CLEAR cs_error.
+
+  IF pv_code = 'COD1'.
+    cv_show_popup = abap_true.
+
+    CALL FUNCTION 'FUNCTION_EXISTS'
```

Comment:
can we refactor the call to FUNCTION_EXISTS + PB_POPUP_PACKAGE_CREATE to a new static method in `lcl_popups`? the 20 lines already exist in this include.